### DISCUSSION TITLE
Why was "at"/"authenticated transfer" chosen?

### DISCUSSION THREAD

**Discussion by Wallby** [bluesky-social/atproto#2105] [Clarifications and "Why?"]
Isn't email also "authenticated transfer", and https as well?

I think the "at:" scheme specifier is pretty neat, but it currently doesn't make much sense to me what it stands for.

Authentication seems not unique to social media and thus not to at.

What does seem unique would be for example accumulative/associative transfer, the former of which refers to that it accumulates through collaboration and the latter of which is a synonym of collaborative[1]. Email is for (always authenticated) sending off and receiving to other users but cannot be edited once submitted as it has no central inbox/storage (thus not accumulative), and http/https is not specifically for interacting with other users (thus not specifically for accumulation, even though it is possible as current social media proves).

[1] https://dictionary.cambridge.org/thesaurus/collaborative

**Comment by snarfed** [comment]
Naming is more art than science. 🤷 😁

**Comment by pfrazee** [comment]
The data repositories are authenticated at-rest via signatures, and this was important enough to the protocol that the early prototypes were named ADX -- the Authenticated Data eXperiment.

*Authenticated Transfer* is perhaps not the ideal name because it skips a couple words -- perhaps a more complete name would be Authenticated Data Transfer Protocol. But, you know, ADTP or ADT is just way less cool than AT, so here we are.

**Reply by bnewbold** [reply]
To further on to Paul's point, both email (with SSL) and HTTP (with SSL) are authenticated *connections*. The content can not be re-distributed by a third party with validation or authenticity. This is a big feature and design point for atproto: content can be mirrored and redistributed through the network, and the final recipient can still independently verify that it came from the indicated source (though it may have gone "stale" in the interim). The fact that you don't have to trust intermediate services like Relays as much (there is still *some* trust involved) is important for our goal of easily switching between service providers, and not having single points of authority for the entire network.

In this way, atproto is closer to git (with signatures).

**Reply by Wallby** [reply]
The technicality of some of what you're both talking about went over my head, but I tried to follow. It seems like you're both saying that the protocol is not specific to social media use, is that correct (can it be used for non social media purpose as well)?

I agree that the AT name is catchy, which I think is important for adoption by not very technical people. Maybe the "at" is sufficient for people to get an idea of what it is for (email is probably the easiest to remember for people because everyone knows what mail is). I wonder though if people won't confuse "at:" and "email:" because from a user's perspective they are both for "sending something towards someone".
